lab-kyle/lib/greet.js

```diff
@@ -0,0 +1,7 @@
+function greeting(name) {
```

Comment:
Solid here as well. What happens if no name is passed? Maybe a good place to throw and error (and test for it).